### PR TITLE
MOTECH-2563: Fixed Admin Queues/Topics 500 response

### PR DESCRIPTION
**Pull request by mkruszynski**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: docs/source/faq.rst [hunk 1]
```diff
@@ -169,3 +169,24 @@ Next, run the integration tests as you would normally do, for example:
     mvn clean install -PIT
 
 Then connect to Tomcat using a remote debugger on port 8000, same as when normally debugging Tomcat.
+
+Why I not see anything in Admin Queues and Topics.
+--------------------------------------------------
```
Comment (llavoie) [inline docs/source/faq.rst]: This should be rephrased to be clearer English, e.g.:

Why am I not seeing anything in "Admin Queues and Topics"?

File: docs/source/faq.rst [hunk 2]
```diff
@@ -169,3 +169,24 @@ Next, run the integration tests as you would normally do, for example:
...
+
+The most likely cause is a RMI connection error. By default RMI is using a random port.
+To set fixed port you have to edit ActiveMQ broker configuration, which is in the file ``/etc/activemq/instances-enabled/main/activemq.xml``.
+Add the following lines to broker configuration, which are you currently using (you can check it in Admin/Settings):
```
Comment (llavoie) [inline docs/source/faq.rst]: "which are you currently using" is not clear -- could you rephrase this?

File: docs/source/faq.rst [hunk 3]
```diff
@@ -169,3 +169,24 @@ Next, run the integration tests as you would normally do, for example:
...
+
+.. code-block:: xml
+
+	<managementContext>
+	    <managementContext createConnector="true" connectorPort="1099" rmiServerPort="1099" />
+	</managementContext>
+
+Next step is ActiveMQ and MOTECH restart.
```
Comment (llavoie) [inline docs/source/faq.rst]: "The next step is to restart ActiveMQ and MOTECH."

File: modules/admin/src/main/resources/webapp/messages/messages.properties [hunk 4]
```diff
@@ -104,8 +104,8 @@ admin.queue.enqueue.count=Enqueue Count
 admin.queue.dequeue.count=Dequeue Count
 admin.queue.expired.count=Expiry Count
 admin.queue.consumer.count=Consumer Count
-admin.queue.error=Could not retrieve queue statistics.Please inform admin.
-admin.queue.message.error=Could not retrieve message information from the queue.Please inform admin.
+admin.queue.error=Could not retrieve queue statistics. Check http://docs.motechproject.org/en/latest/faq.html
```
Comment (llavoie) [inline modules/admin/src/main/resources/webapp/messages/messages.properties]: Can we change this URL to point to an anchor for the specific FAQ questions (rather than the top-level FAQ page)?

File: modules/admin/src/main/resources/webapp/messages/messages.properties [hunk 5]
```diff
@@ -119,7 +119,7 @@ admin.topic.enqueue.count=Enqueue Count
 admin.topic.dequeue.count=Dequeue Count
 admin.topic.expired.count=Expiry Count
 admin.topic.consumer.count=Consumer Count
-admin.topic.error=Could not retrieve topic statistics. Please inform admin.
+admin.topic.error=Could not retrieve topic statistics. Check http://docs.motechproject.org/en/latest/faq.html
```
Comment (llavoie) [inline modules/admin/src/main/resources/webapp/messages/messages.properties]: Same as above

**Comment by sebbrudzinski** [comment]
test this please

File: docs/source/faq.rst [hunk 6]
```diff
@@ -169,3 +169,24 @@ Next, run the integration tests as you would normally do, for example:
     mvn clean install -PIT
 
 Then connect to Tomcat using a remote debugger on port 8000, same as when normally debugging Tomcat.
+
+Why am I not seeing anything in "Admin Queues and Topics"?
+----------------------------------------------------------
+
+The most likely cause is a RMI connection error. By default RMI is using a random port.
+To set fixed port you have to edit ActiveMQ broker configuration, which is in the file ``/etc/activemq/instances-enabled/main/activemq.xml``.
+Add the following lines to broker configuration (name of currently using broker is in ``Admin/Settings/JMX/Broker name``):
```
Comment (sebbrudzinski) [inline docs/source/faq.rst]: the name of the broker in use is in

File: docs/source/faq.rst [hunk 7]
```diff
@@ -169,3 +169,24 @@ Next, run the integration tests as you would normally do, for example:
...
+
+.. code-block:: xml
+
+	<managementContext>
+	    <managementContext createConnector="true" connectorPort="1099" rmiServerPort="1099" />
+	</managementContext>
+
+The next step is to restart ActiveMQ and MOTECH.
```
Comment (sebbrudzinski) [inline docs/source/faq.rst]: Is it really necessary to restart MOTECH? Seems a little odd since we do not touch MOTECH at all in those steps.

File: docs/source/faq.rst (hunk 6 above)
Comment (sebbrudzinski) [inline docs/source/faq.rst]: the broker configuration

File: docs/source/faq.rst [hunk 8]
```diff
@@ -169,3 +169,24 @@ Next, run the integration tests as you would normally do, for example:
...
+Why am I not seeing anything in "Admin Queues and Topics"?
+----------------------------------------------------------
+
+The most likely cause is a RMI connection error. By default RMI is using a random port.
+To set fixed port you have to edit ActiveMQ broker configuration, which is in the file ``/etc/activemq/instances-enabled/main/activemq.xml``.
+Add the following lines to broker configuration (name of currently using broker is in ``Admin/Settings/JMX/Broker name``):
+
+.. code-block:: xml
+
+	<managementContext>
```
Comment (sebbrudzinski) [inline docs/source/faq.rst]: From what I remember, those need to be placed in an alphabetical order. Can you verify this and add a small note about it? It may save someone some time.

Comment (mkruszynski) [inline docs/source/faq.rst]: Indeed, they must be ordered alphabetically.

File: docs/source/faq.rst (hunk 7 above)
Comment (mkruszynski) [inline docs/source/faq.rst]: In my case it helps. Maybe I should wait a little bit more. I'll check it.

**Comment by sebbrudzinski** [comment]
test this please
